tools/ci-build/publisher/src/subcommand/fix_manifests.rs

```diff
@@ -210,6 +233,21 @@ fn update_dep(table: &mut Table, dep_name: &str, versions: &VersionView) -> Resu
     }
 }
 
+// Convert `package_version` into a tilde version requirement
+//
+// For instance, given `package_version` like 0.12.3, the function returns `~0.12`.
+fn convert_to_tilde_requirement(package_version: &str) -> Result<String> {
+    // `package_version` is from the `semver` crate which requires versions to have 3 components,
```

Comment:
There's no type-level enforcement of this assumption. Should the SemVer or Version be passed in directly here rather than a str?

Reply:
I like that and think it's safer in the long run (even if the function is currently private and only called from a call site where `package_version: &str` is derived from `semver::Version`).

Addressed in 49a067d.